src/domain/UBGraphicsWidgetItem.cpp

```diff
@@ -504,9 +504,8 @@ QString UBGraphicsWidgetItem::iconFilePath(const QUrl& pUrl)
     return file;
 }
 
-void UBGraphicsWidgetItem::activeSceneChanged()
+void UBGraphicsWidgetItem::initAPI()
```

Comment:
Still, what would you create a public function called `initAPI` that calls `registerAPI`, rather than changing `registerAPI`'s visibility ? I don't see the interest.

Reply:
OK it makes sense.